### DISCUSSION TITLE
Welcome to pareto-mac Discussions!

### DISCUSSION THREAD

**Discussion by zupo** [ParetoSecurity/pareto-mac#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us how you (plan to) use the Pareto Security app.

**Comment by iampliny** [comment]
Hey! Just wanted to say thank you for making this. 👊 

Look forward to more.
